src/main/java/org/elasticsearch/index/query/GeoBoundingBoxQueryParser.java

```diff
@@ -183,7 +183,7 @@ public Filter parse(QueryParseContext parseContext) throws IOException, QueryPar
         }
```

Comment:
gotcha, I think there are other places where we didn't move the return type to Query, maybe we should do it everywhere?